docs/usage/how-to/cli-mode.mdx

```diff
@@ -17,6 +17,12 @@ for scripting.
 
 ```bash
 pip install openhands-ai
+```
+
+   Or if you prefer not to manage your own Python environment, you can use `uvx`:
+
+```bash
+uvx --python 3.12 --from openhands-ai openhands
```

Comment:
This command will do both the installation AND launch openhands conversation from the command line though. Do we want to say that?